homework/Repository.js

```diff
@@ -12,9 +12,27 @@ class Repository {
    * Render the repository info to the DOM.
    * @param {HTMLElement} container The container element in which to render the repository.
    */
-  render(container) {
-    // TODO: replace the next line with your code.
-    Util.createAndAppend('pre', container, JSON.stringify(this.repository, null, 2));
+  render(container, index) {
+    const repoInfo = [
+      { header: 'Repository', value: this.repository[index].name },
+      { header: 'Description', value: this.repository[index].description },
+      { header: 'Forks', value: this.repository[index].forks },
+      { header: 'Updated', value: this.repository[index].updated_at },
```

Comment:
as the part of saying we don't need the index any more, `this.repository` will be an object instead of an array, means you have to deal with it as an object.